kratos/tests/geometries/test_triangle_3d_3.cpp

```diff
@@ -360,89 +360,89 @@ namespace Testing
     */
     KRATOS_TEST_CASE_IN_SUITE(Triangle3D3ParallelNoIntersection, KratosCoreGeometriesFastSuite) {
         Triangle3D3<Point > triangle_1(
-            GeneratePoint<NodeType >(0.0, 0.0, 0.0),
```

Comment:
actually i think this is an error in the master.

it generates 

       Point(id, x,y,z)

which is a constructor that does not exist...

Reply:
Another copy paste of this bug....

I have also found it in other geometry tests... which was corrected.